site/build.sbt

```diff
@@ -27,6 +27,8 @@ lazy val exerciseSettings: Seq[Setting[_]] =
   )
 
 lazy val clients = Seq(client)
+lazy val doobieVersion = "0.2.3"
+lazy val scalazVersion = "7.1.4"
```

Comment:
scalaz-concurrent already brings in scalaz and so does doobie. We should try to align versions if they aren't already.